adapters/resetdigital/resetdigital.go

```diff
@@ -0,0 +1,301 @@
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		requestBody, err := json.Marshal(splittedRequestData)
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		requests = append(requests, &adapters.RequestData{
+			Method:  "POST",
+			Uri:     a.endpointUri,
+			Body:    requestBody,
+			Headers: getHeaders(requestData),
+			ImpIDs:  []string{imp.ID},
+		})
+	}
+
+	return requests, errors
+}
+
+func processDataFromRequest(requestData *openrtb2.BidRequest, imp openrtb2.Imp, bidType openrtb_ext.BidType) (resetDigitalRequest, error) {
+	var reqData resetDigitalRequest
+
+	if requestData.Site != nil {
+		reqData.Site.Domain = requestData.Site.Domain
+		reqData.Site.Referrer = requestData.Site.Page
+	}
+
+	reqData.Imps = append(reqData.Imps, resetDigitalImp{
+		BidID: requestData.ID,
+		ImpID: imp.ID,
+	})
+
+	if bidType == openrtb_ext.BidTypeBanner && imp.Banner != nil {
+		var tempH, tempW int64
+		if imp.Banner.H != nil {
+			tempH = *imp.Banner.H
+		}
+		if imp.Banner.W != nil {
+			tempW = *imp.Banner.W
+		}
+		if tempH > 0 && tempW > 0 {
+			reqData.Imps[0].MediaTypes.Banner.Sizes = append(
+				reqData.Imps[0].MediaTypes.Banner.Sizes,
+				[]int64{tempW, tempH},
+			)
+		}
+	}
+	if bidType == openrtb_ext.BidTypeVideo && imp.Video != nil {
+		var tempH, tempW int64
+		if imp.Video.H != nil {
+			tempH = *imp.Video.H
+		}
+		if imp.Video.W != nil {
+			tempW = *imp.Video.W
+		}
+		if tempH > 0 && tempW > 0 {
+			reqData.Imps[0].MediaTypes.Video.Sizes = append(
+				reqData.Imps[0].MediaTypes.Video.Sizes,
+				[]int64{tempW, tempH},
+			)
+		}
+	}
+
+	var bidderExt adapters.ExtImpBidder
+	var resetDigitalExt openrtb_ext.ImpExtResetDigital
+
+	if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
+		return resetDigitalRequest{}, err
+	}
+	if err := json.Unmarshal(bidderExt.Bidder, &resetDigitalExt); err != nil {
+		return resetDigitalRequest{}, err
+	}
+	reqData.Imps[0].ZoneID.PlacementID = resetDigitalExt.PlacementID
+
+	return reqData, nil
+}
+
+func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.RequestData, responseData *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if adapters.IsResponseStatusCodeNoContent(responseData) {
+		return nil, nil
+	}
+
+	if err := adapters.CheckResponseStatusCodeForErrors(responseData); err != nil {
+		return nil, []error{err}
+	}
+
+	var response resetDigitalBidResponse
+	if err := json.Unmarshal(responseData.Body, &response); err != nil {
+		return nil, []error{err}
+	}
+
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(request.Imp))
+
+	var errs []error
+	requestImps := make(map[string]openrtb2.Imp)
+	for _, imp := range request.Imp {
+		requestImps[imp.ID] = imp
+	}
+
+	for i := range response.Bids {
+		resetDigitalBid := &response.Bids[i]
+
+		bid, err := getBidFromResponse(resetDigitalBid)
+		// handle the error
+		if bid == nil {
+			// it would be better to return an error here
+			continue
```

Comment:
The comments left here need to be addressed. We discussed returning an error so you might want to change this block to something like:
```
if err != nil || bid == nil {
    errs = append(errs, err)
}